php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml

```diff
@@ -0,0 +1,13 @@
+translation:
+    locales: ["en"]
+    webui:
+        enabled: true
```

Comment:
shouldn't this be enabled only in dev, as you import the routing only in dev ?

Reply:
You are correct. Thank you